libs/core/src/lib/flexible-column-layout/flexible-column-layout.component.html

```diff
@@ -13,10 +13,16 @@
         <ng-container *ngTemplateOutlet="startColumn"></ng-container>
     </div>
     <!-- Left Separator -->
-    <div *ngIf="_leftColumnSeparator" class="fd-flexible-column-layout__separator">
+    <div
+        *ngIf="_leftColumnSeparator"
+        class="fd-flexible-column-layout__separator"
+        i18n-aria-label="coreFlexibleColumnLayout.Left.Separator|Separator aria-label attribute"
+        aria-label="separator"
+    >
         <button
             class="fd-button fd-button--compact fd-button--transparent fd-flexible-column-layout__button"
-            aria-label="Expand"
+            i18n-title="coreFlexibleColumnLayount.Left.Separator.Toogle|Left title attribute"
```

Comment:
please use `@Input` for internationalization
please pay attention to this issue
https://github.com/SAP/fundamental-ngx/issues/6436